src/test_workflow/integ_test/integ_test_runner.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import abc
+
+from manifests.build_manifest import BuildManifest
+from manifests.bundle_manifest import BundleManifest
+from test_workflow.dependency_installer_opensearch import DependencyInstallerOpenSearch
+
+
+class IntegTestRunner(abc.ABC):
+    def __init__(self, args, test_manifest):
+        self.args = args
+        self.test_manifest = test_manifest
+
+        self.bundle_manifest = BundleManifest.from_urlpath("/".join([self.args.path.rstrip("/"), "dist/opensearch/manifest.yml"]))
```

Comment:
This is opensearch specific, shouldn't this be in the parent class(es)?

Reply:
Both runners need OpenSearch. Thus I put it here to avoid duplicated code.

Reply:
I think logically it doesn't make sense: for dashboards this is a dependency. Codewise, this means composition rather than inheritance. If you build another class called "SomethingSomethingOpenSearch", you would add it to "dependencies" and call it accordingly.

Reply:
Created a base class `ServiceStartProperties` to encapsulate such. Code looks much cleaner